syncr_backend/run_backend.py

```diff
@@ -40,12 +88,80 @@ def main():
         nargs='*',
         help='arguments for the function to be ran',
     )
-    args = parser.parse_args()
+    args = parser.parse_args(args=command)
+    return_list[0] = args.function[0]
     if args.args is not None:
-        execute_node_function(args.function[0], args.args)
+        return_list[1] = args.args
     else:
-        execute_node_function(args.function[0], [])
+        return_list[1] = []
+
+
+def execute_function(function_name: str, args: List[str]):
+    """
+    Runs a function with the given args
+    TODO: add real drop/metadata request commands that interface
+    with the filesystem
+    TODO: handle exceptions for real
+
+    :param function_name: string name of the function to run
+    :param args: arguments for the function to run
+    """
+    found_function = False
+    # for functions that create or destroy the init directory
+    if function_name == "node_init":
+        node_init.initialize_node(*args)
+        found_function = True
+    elif function_name == "node_force_init":
+        node_init.force_initialize_node(*args)
+        found_function = True
+    elif function_name == "delete_node":
+        node_init.delete_node_directory(*args)
+        found_function = True
+
+    # drop functions
+    if function_name == "drop_init":
+        drop_init.initialize_drop(args[0])
+        found_function = True
+
+    # request functions, only for debug
+    try:
+        if function_name == "send_drop_metadata_request":
+            print(send_requests.send_drop_metadata_request(
+                args[0],
+                int(args[1]),
+                args[2],
+            ))
+            found_function = True
+        elif function_name == "send_file_metadata_request":
+            print(send_requests.send_file_metadata_request(
+                args[0],
+                int(args[1]),
+                args[2],
+            ))
+            found_function = True
+        elif function_name == "send_chunk_list_request":
+            print(send_requests.send_chunk_list_request(
+                args[0],
+                int(args[1]),
+                args[2],
+            ))
+            found_function = True
+        elif function_name == "send_chunk_request":
+            print(send_requests.send_chunk_request(
+                args[0],
+                int(args[1]),
+                args[2],
+                int(args[4]),
+            ))
+            found_function = True
+    # placeholder only for debugging
```

Comment:
Then get rid of it for merging?

Reply:
the placeholder is when we actually begin testing node interaction, I wanted to make it more convenient to test out specific functions.